### PR TITLE
New Navbar

### PR DESCRIPTION
**Pull request by aayush-05**
1.
image
2.
image
3.
image
4.
image

### CODE REVIEW

File: app/assets/stylesheets/layout.scss [hunk 1]
```diff
@@ -31,66 +31,119 @@ body {
...
+  }
+
+  &.active {
+    color: $teal;
+  }
 }
 
-.logo-bottom {
-  max-height: 80px;
+#navbar-dropdown-1,
```
[the remark this answers is not in the document]
Comment (aayush-05) [reply app/assets/stylesheets/layout.scss:78]: ID selectors works best with Bootstrap dropdowns, can we leave this one ?

**Comment by aayush-05** [comment]
@nitin10s @tachyons please have a look at this

**Comment by nitin10s** [comment]
Is it just me or the Getting Started text colour looks different than the others? Use that lighter shade of black on all the links infact.

**Comment by aayush-05** [comment]
Done @nitin10s

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -1,58 +1,87 @@
-<nav class="navbar navbar-expand-lg navbar-light bg-white">
+<nav class="navbar navbar-expand-lg bg-white navbar-light">
   <a class="navbar-brand" href="/">
-    <%= image_tag("cvlogo.svg", class: "logo", alt: "CircuitVerse Logo") %>
+    <%= image_tag("CircuitVerse.png", class: "navbar-logo", alt: "CircuitVerse Logo") %>
```
Comment (nitin10s) [inline app/views/layouts/_header.html.erb:3]: Why are you not using the new SVG?

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -1,58 +1,87 @@
...
   </button>
-
-  <div class="collapse navbar-collapse" id="collapsedNavbar">
-    <ul class="navbar-nav ml-auto mt-2 mt-lg-0 navigationBar-item">
-      <li class="nav-item px-1">
-        <a class="btn btn-outline-success rounded-0 simulator text-dark " href="/simulator">Simulator</a>
+  <div class="collapse navbar-collapse" id="collapsed-navbar">
+    <ul class="navbar-nav ml-auto navigationBar-item">
+      <li class="nav-item px-1 pt-1">
+        <div class="row-text navbar-search-icon-container">
```
Comment (nitin10s) [inline app/views/layouts/_header.html.erb:12]: This is not navigable by keyboard. Add tabindex="0" and aria-label="search"

File: app/views/layouts/_header.html.erb [hunk 4]
```diff
@@ -1,58 +1,87 @@
...
+          <i class="fa fa-search <%= request.path == '/search' ? 'active' : '' %>" aria-hidden="true"></i>
+        </div>
       </li>
       <li class="nav-item px-1">
-        <a class="nav-link text-dark" href="https://learn.circuitverse.org/">Learn</a>
+        <a class="btn rounded-0 navbar-simulator-text" href="/simulator">Simulator</a>
       </li>
+      <div class="navbar-nav nav-item dropdown">
```
Comment (nitin10s) [inline app/views/layouts/_header.html.erb:19]: This is not navigable by keyboard. Add tabindex="0"

File: app/views/layouts/_header.html.erb (hunk 3 above)
Comment (aayush-05) [reply app/views/layouts/_header.html.erb:15]: Done

File: app/views/layouts/_header.html.erb (hunk 4 above)
Comment (aayush-05) [reply app/views/layouts/_header.html.erb:22]: Added, but while you can traverse through icons and dropdowns using Tab; they can't be activated using keyboard.

**Comment by aayush-05** [comment]
> 1. The colours for nav links are now too light. Use #343a40 for all the links including search icon
> 2. Increase the margins between Search Icon and Simulator.
> 3. The Search Icon Outline looks thick. Can this be resolved?
> 4. For the mobile view, keep the search icon outside the menu collapse.

Done, can't edit the icon. There's a light icon but it requires pro pack

**Comment by ryuk156** [comment]
@aayush-05  you can use svg

**Comment by nitin10s** [comment]
Use the icon from the prototype

File: app/views/layouts/_header.html.erb (hunk 3 above)
Comment (aayush-05) [reply app/views/layouts/_header.html.erb:15]: Mistakenly added; removed now

File: app/views/layouts/_header.html.erb (hunk 4 above)
Comment (aayush-05) [reply app/views/layouts/_header.html.erb:22]: Noted

File: app/views/layouts/_header.html.erb [hunk 5]
```diff
@@ -1,58 +1,95 @@
...
+      </div>
       <li class="nav-item px-1">
-        <a class="nav-link text-dark" href="https://docs.circuitverse.org/">Documentation</a>
-      </li>
-      <li class="nav-item px-1 <%= request.path == '/examples' ? 'active' : '' %>">
-        <a class="nav-link text-dark" href="/examples">Examples</a>
-      </li>
-      <li class="nav-item px-1 <%= request.path == '/contribute' ? 'active' : '' %>">
-        <a class="nav-link text-dark" href="/contribute">Contribute</a>
+        <a class="nav-link navbar-item navbar-text <%= request.path == '/features' ? 'active' : '' %>" href="/features">Features</a>
```
Comment (nitin10s) [inline app/views/layouts/_header.html.erb:32]: Target link for Features nav link should be the Features section on the homepage.

**Comment by satu0king** [comment]
There seems to be a slight positioning issue with the search bar

image

image

**Comment by aayush-05** [comment]
Actually the searchbar you are watching when the search icon is inactive (black) is the one already present on the search page. When the navbar search bar is active, the search bar which is already on the search page is still there but overlapped by the one in navbar. The search bar already present in the search page has to be removed when I revamp this page. (hope this makes sense :p) Shall I do it in this PR only ?

**Comment by satu0king** [comment]
@aayush-05 , I think these 2 links should open in a new tab

**Comment by aayush-05** [comment]
> @aayush-05 , I think these 2 links should open in a new tab
> image

Noted

**Pull request merged**